numpy/_core/numeric.py

```diff
@@ -1799,10 +1799,23 @@ def fromfunction(function, shape, *, dtype=float, like=None, **kwargs):
     ----------
     function : callable
         The function is called with N parameters, where N is the rank of
-        `shape`.  Each parameter represents the coordinates of the array
-        varying along a specific axis.  For example, if `shape`
-        were ``(2, 2)``, then the parameters would be
-        ``array([[0, 0], [1, 1]])`` and ``array([[0, 1], [0, 1]])``
+        `shape`. Each parameter represents the coordinates of the array
+        varying along a specific axis. For example, if `shape` were
+        ``(2, 2)``, then the parameters would be arrays representing
+        coordinates along each axis, like ``array([[0, 0], [1, 1]])``
+        and ``array([[0, 1], [0, 1]])``.
+
+        In the provided examples, `i` and `j` represent coordinates along
```

Comment:
Which examples?

Reply:
Note: the `i` and `j` appear in the examples section of the docstring. Then I think this text belongs there and not in the description of the `function` parameter.

Reply:
Should I created a .txt file so it consists of all examples...is this a good idea.

Reply:
I think this text should remain as-is, and the text refering to the example should be next to the example